kde-apps/kleopatra/kleopatra-9999.ebuild

```diff
@@ -15,10 +15,10 @@ HOMEPAGE="https://www.kde.org/applications/utilities/kleopatra"
 LICENSE="GPL-2+ handbook? ( FDL-1.2+ )"
 KEYWORDS=""
 
-IUSE=""
+IUSE="+kwatchgnupg"
```

Comment:
well since upstream considered the knotifications dep as 'heavy'... admittedly not for regular Linux desktop use. I couldn't yet check what it would change wrt total/implicit dependences, with all other Frameworks already pulled in considered.